internal/command/auth/clusters.go

```diff
@@ -0,0 +1,89 @@
+package auth
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/yaml"
+
+	"github.com/jetstack/jsctl/internal/client"
+	"github.com/jetstack/jsctl/internal/cluster"
+	"github.com/jetstack/jsctl/internal/command/types"
+	"github.com/jetstack/jsctl/internal/config"
+)
+
+func Clusters(run types.RunFunc, apiURL string) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:  "clusters",
+		Args: cobra.ExactArgs(0),
+	}
+
+	cmd.AddCommand(createServiceAccount(run, apiURL))
+
+	return cmd
+}
+
+func createServiceAccount(run types.RunFunc, apiURL string) *cobra.Command {
+	var serviceAccountFormat string
+
+	cmd := &cobra.Command{
+		Use:   "create-service-account [name]",
+		Short: "Create a new service account identity for a cluster",
+		Args:  cobra.MatchAll(cobra.ExactArgs(1)),
+		Long: `jsctl can do this automatically for you, in jsctl clusters connect. However,
```

Comment:
Suggestion: `generate JSS agent credentials for authentication against JSS UI. This is only needed if you are not deploying the agent with jsctl. Output can be json formatted credential or a Kubernetes Secret with the credential for `jetstck-secure` namespace.`

(given that we want to eventually deploy the agent with the operator, we probably don't want to talk about doing it via `jsctl clusters connect` too much..)

Reply:
That's fair enough. I have updated this in 0deda09281664dee71ecea1a74e3aae3ec0bcf8f. However, have adjusted the wording a little.